File: Topic Based Problems/01 Fundamentals/Beginner/16 Swap Two Numbers/Python/SwapTwoNumbersUsingFunction.py
"""
File: SwapTwoNumbersUsingFunction.py
Author: SharifdotG (sharifmdyousuf007@gmail.com)
Description: Swap Two Numbers in Python without using functions
Version: 1.0
Date: 2023-09-24
"""

first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))

print("Before swapping: ")
print("First number:", first_number)
print("Second number:", second_number)

first_number, second_number = second_number, first_number

print("After swapping: ")
print("First number:", first_number)
print("Second number:", second_number)

File: Topic Based Problems/01 Fundamentals/Beginner/16 Swap Two Numbers/C++/SwapTwoNumbersUsingFunction.cpp
/**
 * @file SwapTwoNumbersUsingFunction.cpp
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Swap Two Numbers in C++ using function
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int firstNumber, secondNumber;

    cout << "Enter first number: ";
    cin >> firstNumber;

    cout << "Enter second number: ";
    cin >> secondNumber;

    cout << "Before swapping: " << endl;
    cout << "First number: " << firstNumber << endl;
    cout << "Second number: " << secondNumber << endl;

    swap(firstNumber, secondNumber);

    cout << "After swapping: " << endl;
    cout << "First number: " << firstNumber << endl;
    cout << "Second number: " << secondNumber << endl;

    return 0;
}